chatops/python/docbuilder.py

```diff
@@ -234,7 +234,7 @@ def main():
             except OSError as exception:
                 print_exit('[-] ERR: {0}'.format(exception.strerror),
                            exception.errno)
-        if options['execsummary'] == 'true':  # we're generating a summary as well as a report
+        if options['execsummary'] == True:  # we're generating a summary as well as a report
```

Comment:
I know! *Every time* I see this in other people's code I think: man, I have to do that too. And then I don't. I'm such an imposter. ;)